src/Controllers/IgnoredTicketsController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Aviator\Helpdesk\Controllers;
+
+use Illuminate\Routing\Controller;
+use Aviator\Helpdesk\Repositories\TicketsRepository;
+
+class IgnoredTicketsController extends Controller
+{
+    /** @var array */
+    protected $relations = [
+        'user',
+        'content',
+    ];
+
+    /**
+     * Construct with agents only middleware.
+     */
```

Comment:
This comment isn't right and also isn't really necessary, a constructor is a constructor, you can either omit the docblock entirely or just say Constructor.